src/plugins/mer/mersdkmanager.cpp

```diff
@@ -625,7 +625,16 @@ bool MerSdkManager::generateSshKey(const QString &privKeyPath, QString &error)
 // this method updates the Mer devices.xml, nothing else
 void MerSdkManager::updateDevices()
 {
+    if (m_sdks.count() == 0)
```

Comment:
Consider `isEmpty()` (according to Qt convention for better readability)

Reply:
Fixed